e3sm_diags/plot/cartopy/zonal_mean_2d_stratosphere_plot.py

```diff
@@ -1,15 +0,0 @@
-import xarray as xr
```

Comment:
This file is not needed anymore (`zonal_mean_2d_plot.py` is used directly by this set)